client/my-sites/customer-home/main.jsx

```diff
@@ -64,8 +71,21 @@ const Home = ( {
 				subHeaderText={ translate( 'Your home base for posting, editing, and growing your site.' ) }
 				align="left"
 			/>
-			<div className="customer-home__view-site-button">
-				<Button href={ site.URL } onClick={ trackViewSiteAction }>
+			<div className="customer-home__heading-actions">
+				{ isStaticHomePage && (
+					<Button
+						className="customer-home__edit-site-button"
+						href={ editHomePageUrl }
+						onClick={ trackEditSiteAction }
+					>
+						{ translate( 'Edit site' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Edit Site', 'NULL', { context: 'NULL'} )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/home-heading-edit-site-button)